ci-operator/config/jboss-fuse/camel-k-test-container/jboss-fuse-camel-k-test-container-main__camel-k-ocp4.14-lp-rosa.yaml

```diff
@@ -33,22 +33,24 @@ tests:
     env:
       FIREWATCH_CONFIG: |
         [
-          {"step": "aws-*", "failure_type": "all", "classification": "Infrastructure AWS", "jira_project": "LPTOCPCI", "group": {"name": "cluster", "priority": 1}},
-          {"step": "rosa-*", "failure_type": "all", "classification": "Infrastructure Provisioning - Cluster", "jira_project": "LPTOCPCI", "group": {"name": "cluster", "priority": 1}},
-          {"step": "install-operators", "failure_type": "all", "classification": "Infrastructure Provisioning - Operators", "jira_project": "LPTOCPCI", "group": {"name": "operator", "priority": 1}},
-          {"step": "gather-*", "failure_type": "pod_failure", "classification": "Other", "jira_project": "LPTOCPCI", "group": {"name": "cluster", "priority": 2}},
-          {"step": "gather-*", "failure_type": "test_failure", "classification": "NONE", "jira_project": "NONE", "ignore": "true"},
-          {"step": "openshift-*", "failure_type": "all", "classification": "Other", "jira_project": "LPTOCPCI", "group": {"name": "cluster", "priority": 2}},
-          {"step": "camel-k-interop-tests", "failure_type": "test_failure", "classification": "Test Failure", "jira_project": "CMLK", "group": {"name": "operator", "priority": 2}}
+          {"step": "aws-*", "failure_type": "all", "classification": "Infrastructure AWS", "jira_project": "LPTOCPCI", "group": {"name": "cluster", "priority": 1}, "jira_additional_labels": ["4.14-lp","rosa-lp","camel-k-lp"]},
```

Comment:
As I'm building out the PR for rosa-hypershift testing it hit me that we should probably place this firewatch config in alphabetical order so that the step we are searching for is more easily findable.

This also will follow the pattern that these steps are shown to us in the artifacts of the run
https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_release/45508/rehearse-45508-periodic-ci-jboss-fuse-camel-k-test-container-main-camel-k-ocp4.14-lp-rosa-camel-k-interop-aws-rosa-sts/1724006133703643136/artifacts/camel-k-interop-aws-rosa-sts/

Reply:
@ascerra I thought about it and want to sync with you regarding my thoughts (I agree & have suggestions), so lets keep it outside of this PR for now if it's ok

Reply:
Yes I am ok with keeping it out of this PR, there are plenty more coming for rosa classic so handling it in another PR works for me.